libraries/shared/storage/backfill/storage_value_loader_test.go

```diff
@@ -103,7 +119,7 @@ var _ = Describe("StorageValueLoader", func() {
 		runnerErr := runner.Run()
 		Expect(runnerErr).NotTo(HaveOccurred())
 		Expect(headerRepo.GetHeadersInRangeStartingBlock).To(Equal(blockOne))
-		Expect(headerRepo.GetHeadersInRangeEndingBlock).To(Equal(blockTwo))
+		Expect(headerRepo.GetHeadersInRangeEndingBlock).To(Equal(blockThree))
```

Comment:
What happened to blockTwo?

Reply:
I added another block so now the range is 1-3 instead of 1-2